e2e_tests/integration/0.index.spec.ts

```diff
@@ -45,13 +46,59 @@ describe('Neo4j Browser', () => {
       .contains(':server connect')
       .click()
     cy.typeInFrame(':play movies{enter}', 0)
-    cy.get('[data-testid=frame]').contains('the Bacon Path')
+    cy.get('[data-testid="frame"]').contains('the Bacon Path')
   })
+
   it('can connect', () => {
     const password = Cypress.config('password')
     cy.connect('neo4j', password)
   })
 
+  it('can show Canny badge in "Document" icon and correspond side bar drawer', () => {
+    // The Canny badge should appear when initially connected to the database
```

Comment:
what I worry about with these tests are that they'll fail if canny is unavailable/tests run without network. Also not sure if we can run them twice, since the changelog has already been read the second time? Not a super big problem per se, but it could be really annoying when debugging

Reply:
In the general case I prefer end2end tests for this kind of thing, but because of the canny-issues I just mentioned I think these could work better as unit tests of App.tsx where we mock the 'https://canny.io/sdk.js' url and see that it gets called.

Reply:
@OskarDamkjaer - do you think it's better to remove those test cases? Or add some conditions that we only run it when the network and Canny is available? Probably the former option is more straightforward?

Reply:
I expanded on this but forgot to send the comment! I generally like e2e tests for these kinds of changes, but I was thinking it'd be more practical to do a unit test in this case. I'm thinking if we write a unit test for App.tsx where the we mock url for the canny sdk and ensure that it gets pinged we'd have confidence that the bug you fixed doesn't come back